src/generated_test/resources/assets/data_gen_test/sounds.json

```diff
@@ -0,0 +1,125 @@
+{
+  "ambient.underwater.loop.additions": {
+    "sounds": [
+      "minecraft:ambient/underwater/additions/bubbles1",
```

Comment:
it would be nice if this could match the vanilla file, so if you could auto-remove the minecraft prefix if that would be great!